lib/helper.js

```diff
@@ -167,18 +168,60 @@ async function listBuildClustersByGroup() {
     return allBuildClustersByGroup;
 }
 
+/**
+ * check if pipeline is authorized to use the build cluster
+ * @param {String} pipelineName the name of pipeline
+ * @param {Object} buildCluster the build cluster
+ * @return {Boolean}
+ */
+function checkScmOrgPermission(pipelineName, buildCluster) {
```

Comment:
tiny suggestion: maybe a more boolean-sounding function name like `hasClusterAccess`
```suggestion
function hasClusterAccess(pipelineName, buildCluster) {
```